modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -228,95 +232,182 @@ For more information, see <<clustering-discovery-k8s>>.
 In-place rolling reduces fault tolerance temporarily because you are restarting a running server.
 To keep fault-tolerance, you can introduce a fourth server temporarily.
 ====
-. Ensure that the new settings are added to the configuration as detailed in the preparation section.
+. For all the servers, ensure that new settings are added to the configuration as detailed the xref:clustering/setup/discovery.adoc#_preparation[Preparation] section.
++
```

Comment:
Rogue `+` signs.

Reply:
Yeah, there are quite a few of those. We don't need a `+` sign before a step, aka `. `.